tests/test_layers.py

```diff
@@ -1,8 +1,14 @@
 import unittest
 import ddt
 import json
+import sys
+
+sys.path.append("/home/alex/fpgaconvnet-model-rebase/")
+sys.path.append("../fpgaconvnet/")
```

Comment:
I kind of dont like to explicitly add things to the python path this way.
I would suggest to find a better way to import what is needed from the fpgaconvnet folder

Reply:
Yeah it was on accident, I'll remove it.